lib/fluent/plugin/in_opensearch.rb

```diff
@@ -215,7 +215,7 @@ def reachable_host?(host)
         host: ["#{host[:scheme]}://#{host[:host]}:#{host[:port]}"],
         user: host[:user],
         password: host[:password],
-        reload_connections: @reload_connections,
+        # reload_connections: @reload_connections,
```

Comment:
Hey, could you describe why this line is commented out?
Would you mind if you revive this line?